apps/backend/src/db/migrations/20250813100759_update_route_categories_and_add_categories_table.ts

```diff
@@ -0,0 +1,69 @@
+import { type Knex } from "knex";
+
+const TableName = {
+	CATEGORIES: "categories",
+	ROUTE_CATEGORIES: "route_categories",
+	ROUTES: "routes",
+};
+
+const ColumnName = {
+	CATEGORY_ID: "category_id",
+	CREATED_AT: "created_at",
+	ID: "id",
+	KEY: "key",
+	NAME: "name",
+	ROUTE_ID: "route_id",
+	UPDATED_AT: "updated_at",
+};
+
+const MAX_NAME_LENGTH = 32;
```

Comment:
i would make limit larger
```suggestion
const MAX_NAME_LENGTH = 100;
```